docker/README.md

```diff
@@ -17,12 +17,3 @@ PLATFORM="virtual" ./docker/run-dev.sh
 ```
 
 Both scripts accept different variables for customization. Please refer to the corresponding scripts for the full list of available variables to use.
-
-## Build and push docker images to a container registry
-
-After building a docker image, you can push it to a container registry (e.g., Azure Container Registry). You can use the `push_image.sh` script to automatically build and push the image to ACR:
-
-```bash
-# Build docker image for SGX
-./docker/push_image.sh
-```
```

Comment:
Why removing this entire section?

Reply:
This whole page needs to be updated and the building of an image alone does not help. And, I am not even sure if this is the right way to build anymore so it needs to be rewritten anyway.

Reply:
This section is just for pushing an image to a container registry and it can be useful for local development and testing. Unless there's a specific reason to remove it (I don't see anything SGX-related in here, except for the comment line), I would not remove it.

Reply:
Oh wait sorry I thought you were referring to another page. This piece of code and docs is not used anymore, anyone can push the docker image to their registry and there is no need to have a script to do that. The less we have to maintain the better